includes/functions.php

```diff
@@ -580,6 +580,7 @@ function createHost($host, $community = NULL, $snmpver, $port = 161, $transport
         'port' => $port,
         'transport' => $transport,
         'status' => '1',
+        'ip' => $ip,
```

Comment:
ip is a varbinary field:

| ip                        | varbinary(16)                                | NO   |     | NULL    |                |

Check poll_device() function:

$db_ip = inet_pton($ip);